generate/base.py

```diff
@@ -142,13 +142,20 @@ def main(
     fabric.print(f"Time to instantiate model: {time.perf_counter() - t0:.02f} seconds.", file=sys.stderr)
 
     t0 = time.perf_counter()
-    with lazy_load(checkpoint_path) as checkpoint:
```

Comment:
If this approach is acceptable to support FSDP with meta-device initialization in the generate scripts, I'll add it to the others too.